sandy/sections/mf8.py

```diff
@@ -288,7 +295,9 @@ def _read_rdd(tape, mat):
             # radiation types
             "DE": L.B[1::2],
             # decay constant in 1/s, 0 if stable
-            "LAMBDA": math.log(2.0)/L.C1 if L.C1 else 0
+            "LAMBDA": math.log(2.0)/L.C1 if L.C1 else 0,
+            # uncertainty on decay constant
+            "DLAMBDA": math.log(2.0)*L.C2/L.C1**2 if L.C1 else 0
```

Comment:
PEP8

Reply:
with this I mean that you should write `math.log(2.0) / L.C1` instead of `math.log(2.0)/L.C1`